internal/feedback/feedback.go

```diff
@@ -82,6 +82,55 @@ func (cf ConstraintFeedback) LogFeedback(logger *log.Logger) {
 	}
 }
 
+type brokenImport struct {
+	projectPath string
+	version     *gps.StringDiff
+	revision    *gps.StringDiff
+}
```

Comment:
This struct is only tracking versions and revisions for feedback, but that may not be sufficient.  Not sure...

Reply:
We want to warn when one of the following properties of a locked project changes:

* source 
* branch
* version
* revision